### DISCUSSION TITLE
V4版本没办法按V3的命令安装系统服务了吗？

### DISCUSSION THREAD

**Discussion by jayolim** [IrineSistiana/mosdns#394] [Q&A] [answered]
V3版本可以使用该命令mosdns -s install -dir /etc/mosdns && mosdns -s start
完成安装和重启后自启动工作指定目录。
现V4版本wiki中的mosdns start -c config_file -d working_dir​可以启动但无法像之前的版本运行在后台和开机自启。
系统是LXC Debian

**Comment by IrineSistiana** [comment, marked as answer]
命令换了 看 mosdns -h

**Reply by jayolim** [reply]
谢谢作者大大，发现新版Wiki已经写有命令成功配置了。
